CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+# Code of Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other such characteristics.
+
+Everyone is expected to follow the [Scala Code of Conduct] when discussing the project on the available communication channels. If you are being harassed, please contact us immediately so that we can support you.
+
+## Moderation
+
+Any questions, concerns, or moderation requests please contact a member of the project.
+
+- Alexandru Nedelcu: [gitter](https://gitter.im/alexandru) | [twitter](https://twitter.com/alexelcu) | [email](mailto:coc@temp18.alexn.org)
```

Comment:
Oh, this reminds me to update my email address, good thing I took a look. Please update to:

```
y20+coc@alexn.org
```

In public I like publishing dynamic email addresses and to change them, to combat spam. Unfortunately I have to keep track of where my address ends up at for when I change it 🙂